committees/config.py

```diff
@@ -0,0 +1,82 @@
+from importlib import import_module
+
+from django.apps import apps
+
+
+class CommitteeConfig:
+    """ Configurations for additional tabs on committee pages """
+    url_keyword = None
+    name = None
+    tab_select_keyword = None
+    url_name = None
+
+    """ Namespace for url. Can be left none. If not left none, know that url navigation will go like:
+    committees:<namespace>:url_name
+    """
+    namespace = None
+
+    def get_urls(self):
+        """ Builds a URLconf.
+        When defining the view classes make sure they implement this config. Eg:
+        > path('url_path/', views.MyCommitteeTabView.as_view(config=MyCommitteeConfig))
+        This way the AssociationGroupMixin checks if this tab is accessable for the particular group.
+        """
+        raise NotImplementedError
+
+    @property
+    def urls(self):
+        return self.get_urls(), self.namespace, self.namespace
+
+    def get_tab_data(self):
+        """ This creates the data for tab display
+        name: The display name
+        tab_select_keyword: value that marks that this tab is selected
+        url_name: name of the url. Url may only accept group_id as a kwarg.
+        """
+        return {
+            'name': self.name,
+            'tab_select_keyword': self.tab_select_keyword,
+            'url_name': self.url_name,
+        }
+
+    @classmethod
+    def is_valid_for_group(cls, association_group):
+        """
+        Checks for a given association_group is the group has access to this tab page.
+        :param association_group: The association group
+        :return: Boolean determining access
+        """
+        return True
+
+    def get_local_quicklinks(self, association_group):
```

Comment:
It took me a while to figure out, but these are the extra urls that are added on a group's overview page, and are currently only used for groups that have permission to modify said items.

The links themselves are fine, but the pages they lead to have major overlap with the "inventory" tab. This is fine for committees without permissions to edit (the tab allows them to view their owned items), but not so much for committees with do have this permission as there's now two pages which allow the same thing (being item links), but one page (the catalogue) allows editing more stuff.

It also seems that I reach a 403 if I attempt to edit an item as a non-admin.